modules/serverless-rn-1-29-0.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29
+
+{ServerlessProductName} 1.29 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.8.
+* {ServerlessProductName} now uses Knative Eventing 1.8.
+* {ServerlessProductName} now uses Kourier 1.8.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.8.
+* {ServerlessProductName} now uses Knative Kafka 1.8.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+** The latest release is available through the `latest` channel.
+** Releases older than the latest are available through the version-based channels.
++
+To use these, update the channel parameter in the subscription object YAML file from `stable` to either `latest` or a version-based channel, such as `Serverless 1.28`.
++
+This change allows you to receive updates not only for the latest release, but also for releases in the Maintenance phase.
+
+* You can now create {ServerlessProductName} functions through DevConsole using {product-title} Pipelines.
+
+* A `PodDistruptionBudget` and a `HorizontalPodAutoscaler` have been added for the `3scale-kourier-gateway` deployment.
+** `PodDistruptionBudget` is used to define the minimum availability requirements for pods in a deployment.
+** `HorizontalPodAutoscaler` is used to automatically scale the number of pods in the deployment based on demand or on your custom metrics.
+
+* API version `v1alpha1` of the Knative Operator Serving and Eventings CRDs has been removed. You need to use the `v1beta1` version instead. This does not affect the existing installations, because CRDs are updated automatically when upgrading the Serverless Operator.
+
+* FIXME Is this actually GAing in 1.29? (Multi-container support for Knative Serving is now generally available (GA). This feature allows you to use a single Knative service to deploy a multi-container pod.)
```

Comment:
I'm also not sure if it is GA in 1.29 or not - https://issues.redhat.com/browse/SRVKS-1012 
cc @skonto

Reply:
It is going to be 1.30 see https://redhat-internal.slack.com/archives/CD87JDUB0/p1685454728073959